src/stories/purchase-flow-custom-appearance-desktop-embedded.stories.svelte

```diff
@@ -91,7 +91,6 @@
 <Story
   name="Email Input"
   args={{ currentView: "needs-auth-info", isInElement: true }}
-  parameters={{ viewport: { defaultViewport: "desktop" } }}
```

Comment:
These were causing unwanted behaviour (setting as mobile also when the intended viewport was supposed to be desktop by clicking on a mobile story first)

Reply:
so how are you forcing the viewport to desktop now?